taker-frontend/src/components/Trade.tsx

```diff
@@ -93,19 +99,22 @@ export default function Trade({
     let outerCircleBg = useColorModeValue("gray.100", "gray.700");
     let innerCircleBg = useColorModeValue("gray.200", "gray.600");
 
-    const referencePrice = `$${referencePriceAsNumber?.toLocaleString() || "0.0"}`;
-    const askPrice = `$${askPriceAsNumber?.toLocaleString() || "0.0"}`;
-    const liquidationPrice = `$${liquidationPriceAsNumber?.toLocaleString() || "0.0"}`;
+    // Use `Number` wrapper here to ensure the "," separator is printed.
+    const askPrice = `$${Number(askPriceAsNumber)?.toLocaleString() || "0.0"}`;
```

Comment:
At this point it would be good to introduce components for this stuff and add tests :)